### PR TITLE
Baldur/adds before and after verbiage

### PR DESCRIPTION
**Pull request by baldur**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,9 +6,11 @@ buildscript {
 
     dependencies {
         classpath 'org.gradle.api.plugins:gradle-nexus-plugin:0.2'
+        classpath 'eu.appsatori:gradle-fatjar-plugin:0.2-rc1'
     }
 }
 
+apply plugin: 'fatjar'
```
Comment (ecgreb) [inline build.gradle:13]: Why do we need this? Is this so 'org.json:json:20131018' is included in the jar?

Comment (baldur) [inline build.gradle:13]: this is for the tester app. It's a seperate task so it shouldn't leak into our app

Comment (baldur) [inline build.gradle:13]: @ecgreb just double checked the regular install produces lighter jar

 % unzip geojson-0.1-SNAPSHOT.jar 
Archive:  geojson-0.1-SNAPSHOT.jar
   creating: META-INF/
  inflating: META-INF/MANIFEST.MF  
   creating: com/
   creating: com/mapzen/
   creating: com/mapzen/osrm/
  inflating: com/mapzen/osrm/RouteHelper.class  
  inflating: com/mapzen/osrm/Instruction.class  
  inflating: com/mapzen/osrm/Route.class  
   creating: com/mapzen/geo/
  inflating: com/mapzen/geo/GeoFeature.class  
  inflating: com/mapzen/geo/Geometry.class  
  inflating: com/mapzen/geo/DistanceFormatter.class

Comment (ecgreb) [inline build.gradle:13]: lgtm.

**Comment by ecgreb** [comment]
@baldur I don't see any use of the route fixtures in this project. Are they used in app tests and/or snap tester?

**Comment by baldur** [comment]
@ecgreb https://github.com/mapzen/simple-geojson/blob/master/src/test/java/com/mapzen/osrm/RouteTest.java#L137

**Comment by ecgreb** [comment]
aha.. this was outside the power of Alt + F7 ;)

**Pull request merged**